### PR TITLE
Set default size of VAudioTrack with a computed property

### LINKED ISSUE #721: All content grid is using an invalid size prop
## Description

The `Box` audio layout has 3 possible `size` values (`s`, `m`, `l`), but the All content grid is using `full`. This doesn't show visually, but causes warnings in the console.

## Reproduction

1. Run the app and search for 'All content'
2. See warning in the console.
```
[Vue warn]: Invalid prop: custom validator check failed for prop "size". 
found in
---> <VAudioTrack> at src/components/VAudioTrack/VAudioTrack.vue

```

## Expectation

Component should use a valid prop, and should not cause warnings.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by krysal**
## Fixes

Fixes #721 by @obulat 

## Description

This PR makes the default size for the boxed layout `null` instead of `'m'` (preserved for all other layouts if not explicitly specified) with a computed property, this way we avoid the warning for an invalid prop and it still looks good in the All search results grid, as no fixed `width` is applied.

## Testing Instructions

Follow instructions on the related issue and observe no more warnings are shown, maintaining the normal aspect of the all search results grid.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [-] I added or updated tests for the changes I made (if applicable).
- [-] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by obulat — APPROVED** [review]
Thank you for fixing this, it wasn't a trivial change.
It might be a good idea to add/edit comments, because the `VBoxLayout` has a comment at the top saying `<!-- Should be wrapped by a fixed width parent -->`, which probably isn't true anymore?

**Comment by zackkrida** [comment]
`<!-- Should be wrapped by a fixed width parent -->`

Yes @obulat maybe something like "The width is determined by the parent element" or something would be clearer.

**Comment by krysal** [comment]
Thanks for noticing it, comment updated 📝

File: src/components/VAudioTrack/layouts/VBoxLayout.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div :style="{ width }">
-    <!-- Should be wrapped by a fixed width parent -->
+    <!-- The width is determined by the parent element is the 'size' property is not specified -->
```
Comment (zackkrida) [inline src/components/VAudioTrack/layouts/VBoxLayout.vue:3]:
```suggestion
    <!-- The width is determined by the parent element if the 'size' property is not specified. -->
```

**Review by zackkrida — APPROVED** [review]
LGTM! I just committed a small typo fix in the new comment 👍

File: src/components/VAudioTrack/layouts/VBoxLayout.vue (hunk 1 above)
Comment (obulat) [inline src/components/VAudioTrack/layouts/VBoxLayout.vue]:
```suggestion
    <!-- The width is determined by the parent element if the 'size' property is not specified -->
```

**Pull request merged**